code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -860,6 +860,7 @@
 					return
 
 				thralls.remove_antagonist(T.mind, 0, 0)
```

Comment:
think it might be better to add `QDEL_NULL(vampire)` to thralls' remove antagonist
![image](https://user-images.githubusercontent.com/22774890/115049313-e19aa680-9eda-11eb-82e2-2cf486db4e16.png)

Reply:
Discussed this over discord, apparently `QDEL_NULL(vampire)` didn't work for some reason but the current thing was moved over to the requested file.